### PR TITLE
Java: update docs and videos

### PR DESCRIPTION
**Pull request by Eskibear**
Signed-off-by: Yan Zhang <yanzh@microsoft.com>

### CODE REVIEW

**Comment by Eskibear** [comment]
@nickzhums please take a look.
@jdneo please help to update `testng.mp4` with latest UX.

**Comment by jdneo** [comment]
Thanks for the reminder

File: docs/languages/java.md [hunk 1]
```diff
@@ -53,35 +53,31 @@ Alternatively, you can add Java language support to VS Code by installing the po
...
-![Java Extensions](images/java/extensions.png)
-
 This document describes some of the key features included in those Java extensions.
 
 ## Getting started
 
 **NOTE:** If you are using VS Code on Windows and want to take advantage of the Windows Subsystem for Linux, see [Developing in WSL](/docs/remote/wsl).
 
-Before you start, you must have the Java SE Development Kit (JDK) on your local environment. To run the Java Language Support (redhat.java) extension itself, Java SE 11 or above version is required; for projects, the extension supports version 1.5 or above. For how to configure these versions, refer to [Configure JDK](/docs/java/java-project.md#configure-jdk).
+Before you start, you must have the Java SE Development Kit (JDK) on your local environment. To run the Language Support for Java™ (redhat.java) extension itself, Java SE 11 or above version is required; for projects, the extension supports version 1.5 or above. For how to configure these versions, refer to [Configure JDK](/docs/java/java-project.md#configure-runtime-for-projects).
```
Comment (nickzhums) [inline docs/languages/java.md:75]: we will modify this part once we release embedded JDK right?

File: docs/languages/java.md [hunk 2]
```diff
@@ -53,35 +53,31 @@ Alternatively, you can add Java language support to VS Code by installing the po
...
 
-For developers new to Java or new to VS Code, we provide a Getting Started experience. Once you've installed the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), you can open the Getting Started experience from within VS Code with the **Java: Getting Started** command from the Command Palette.  Open the Command Palette (`kb(workbench.action.showCommands)`) and type "Java: Getting Started".
+For developers new to Java or new to VS Code, we provide some tips. Once you've installed the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), you can open the tips from within VS Code with the **Java: Tips for Beginners** command from the Command Palette.
```
Comment (nickzhums) [inline docs/languages/java.md:77]:
```suggestion
For developers new to Java or new to VS Code, we do provide some tips in our extensions. Once you've installed the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), you can see the tips using the **Java: Tips for Beginners** command from the Command Palette in VS Code
```

**Comment by nickzhums** [comment]
overall lgtm - just left minor comments - thanks!

File: docs/languages/java.md (hunk 1 above)
Comment (Eskibear) [reply docs/languages/java.md:75]: Yes. 
cc  @testforstephen , we would better create an issue, tracking a todo list after we release platform-specific extensions.

Comment (testforstephen) [reply docs/languages/java.md:75]: Here is the todo list.
https://github.com/microsoft/vscode-java-pack/issues/845

**Comment by jdneo** [comment]
PR for updating the testing video: https://github.com/microsoft/vscode-docs/pull/5004

**Pull request merged**